src/uart_comm_win.c

```diff
@@ -841,4 +841,9 @@ void uart_process_timeout(struct uart *port)
     }
 }
 
+int uart_get_rs485_config(struct uart *port, struct uart_config *config)
+{
+    // Unsupported on Windows
```

Comment:
```suggestion
    // Unsupported on Windows
    (void) port;
    (void) config;
```

I think that this should give compiler warnings on Windows with this.